api/v1beta2/cloudstackaffinitygroup_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```

Comment:
This is not updated at all. I think you can keep 2022.

Reply:
I keep getting 
`Error: pkg/webhookutil/webhook_validators.go:1:11: Expected:2023, Actual: 2022 The Kubernetes Authors. (goheader)` and similar error even this PR does not change those files. 

I created a PR to https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/pull/209 to address above error.

Reply:
Would it work if you change it to `Copyright 2022-2023`?  That would be more accurate.

Reply:
I answered this question on PR https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/pull/209